Cesium3DTilesSelection/src/TilesetContentManager.h

```diff
@@ -145,6 +149,7 @@ class TilesetContentManager
   int32_t _tileLoadsInProgress;
   int32_t _loadedTilesCount;
   int64_t _tilesDataUsed;
+  Tile::LoadedLinkedList& _loadedTiles;
```

Comment:
This is dangerous. The `TilesetContentManager` can and sometimes does outlive the `Tileset`. So by having this reference here, we need to be careful that we'll never try to use it after the `Tileset` has been destroyed.